exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcMetricExporter.java

```diff
@@ -40,8 +43,8 @@
    * Creates a new Jaeger gRPC Metric Reporter with the given name, using the given channel.
    *
    * @param channel the channel to use when communicating with the Jaeger Collector.
```

Comment:
Still says Jaeger in here, FYI.

Reply:
do you want to update the javadoc while you're in here, updating other javadoc?  ;)

Reply:
Better do it now before this slips through again ;) Done in d013f7b